tests/wpt/mozilla/meta/mozilla/webgl/tex_image_2d_canvas_no_context.html.ini

```diff
@@ -1,4 +0,0 @@
-[tex_image_2d_canvas_no_context.html]
```

Comment:
@jdm that one is deleted again, this time using the server logs, should I revert back?